src/main/java/org/grizz/model/Planet.java

```diff
@@ -1,10 +1,19 @@
 package org.grizz.model;
 
+import com.google.common.collect.Sets;
 import lombok.Builder;
 import lombok.Data;
 
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.*;
+
 @Data
 @Builder
+@Document(collection = "planets")
 public class Planet {
+
     private String id;
```

Comment:
Maybe we should add separated fields: id and planetName? Id will be as a db id and planetName will be from player? In Controller we add method findByName.